config/deploy/docker/vault_poll_state

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Why are we deleting this?

Reply:
I was syncing from the warehouse. Maybe it's not over there? I'll bring it back in.